modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/http/ESHttpClient.java

```diff
@@ -53,10 +53,11 @@
 import java.security.KeyStoreException;
 import java.io.FileInputStream;
 import org.apache.log4j.PropertyConfigurator;
+import org.syslog_ng.LogThrDestDrvConsts;
 
 import java.io.IOException;
 
-public class ESHttpClient implements ESClient {
+public class ESHttpClient implements ESClient, LogThrDestDrvConsts {
```

Comment:
By replacing `LogThrDestDrvConsts` with a statically imported enum class, you can remove this extra interface.

Reply:
I resolve this, because in the end I could eliminate the whole class.